README.md

```diff
@@ -42,23 +42,14 @@ TEMPLATES = [
 ```
 If you choose not use it as a built-in, you will need to add `{% load suspense %}` to the top of your template whenever you want to use suspense.
 
-### 2. Add `suspense` to `urls.py`:
-This is necessary because some parts of the pages will be loaded asynchronously (via http).
-```python
-from django.urls import include, path
-
 
-urlpatterns = [
-    ...,
-    path('/_suspense/', include('suspense.urls'))
-]
-```
-
-### 3. Create view with slow lazy load object:
+### 2. Create view with slow lazy load object:
 Because django executes database queries lazily, they may sometimes not work as expected. Let's try to create a very slow but lazy object and write a view function:
 ```python
+from suspense.shortcuts import render
+
 # app/views.py
-def view(request):
+async def view(request):
```

Comment:
That seems fine, but as described in PR Description, there should probably be a mention of the breaking change above in the readme:

> It requires to use an ASGI server because of the async nature of the implementation.